src/renderer/components/ExportCard.tsx

```diff
@@ -18,7 +18,7 @@ interface Props {
 const ExportCard = ({ progress }: Props) => {
   return (
     <Card>
-      <Typography variant="h-300" color="primary.main">
+      <Typography variant="body1" color="primary.main">
```

Comment:
Thanks for the fix. But please change to h3.

Reply:
Ah yes

Reply:
Changed